docs/Changelog

```diff
@@ -14,6 +14,10 @@ Stable versions
 	- Fix out-of-bounds reads in His Master's Noise Mupp instruments.
 	- Fix slow ProWizard testing.
 	- Replace rand() with a built-in reentrant alternative.
+	- xmp_set_tempo_factor now returns -XMP_ERROR_STATE when called prior
+	  to xmp_start_player (instead of causing crashes).
+	- Fix mixer crashes caused by previously valid tempo factors after
+	  sample rate or BPM changes.
```

Comment:
These need to go into lite/ChangeLog too.

And we don't have any information about sample rate or BPM changes in there.

Reply:
OK, I duplicated these lines into lite/Changelog.